src/main/java/frc/robot/RobotContainer.java

```diff
@@ -160,7 +154,7 @@ private void configureBindings(){
 //        buttonHelper.createButton(9, 1, new AutoAlignCommand(drivetrainSubsystem, autoBuilder, AutoConstants.ALIGNMENT_POSITION.SINGLE_STATION), RunCondition.WHEN_PRESSED);
 
         MultiButton rightJoystickButton = buttonHelper.createButton(12);
-        buttonHelper.createButton(12, 0, new DriverArmPositionCommand(armSubsystem, elevatorSubsystem, rightJoystickButton), RunCondition.WHEN_PRESSED);
+        buttonHelper.createButton(12, 0, new DriverArmPositionCommand(armSubsystem, elevatorSubsystem, rightJoystickButton).beforeStarting(new StartEndCommand(() -> elevatorSubsystem.setPosition(ElevatorConstants.ElevatorPositions.DOWN), elevatorSubsystem::stop).until(elevatorSubsystem::motionFinished)), RunCondition.WHEN_PRESSED);
```

Comment:
This should probably use the ElevatorDownCommand and have the unless decorator. That way it won't run the elevator down when the elevator is already down.